### PR TITLE
OSSM-4779: spec section appears twice in the SMCP yaml sample

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSSM-4779

Link to docs preview: https://64260--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-observability#ossm-integrating-with-user-workload-monitoring_observability

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by monroe-dylan-m — APPROVED** [review]
Lgtm!

File: modules/ossm-integrating-with-user-workload-monitoring.adoc [hunk 1]
```diff
@@ -86,7 +86,6 @@ spec:
       enabled: false # <2>
     kiali:
       name: kiali-user-workload-monitoring
-spec:
   meshConfig:
     extenstionProviders:
```
Comment (luksa) [inline modules/ossm-integrating-with-user-workload-monitoring.adoc:90]: Typo:
```suggestion
    extensionProviders:
```

Comment (rh-tokeefe) [reply modules/ossm-integrating-with-user-workload-monitoring.adoc]: Eagle Eyes @luksa! Thanks Marko, I only looked at the line called out in the story and didn't even see this in the next line. Appreciate your diligence!

Change has been submitted.

**Review by luksa — APPROVED** [review]
LGTM

**Review by gwynnemonahan — APPROVED** [review]
LGTM

**Comment by rh-tokeefe** [comment]
Rebased PR on Sept 7

**Comment by bscott-rh** [comment]
Merge review complete. @rh-tokeefe you have the "do not merge" label on this PR. Would you like me to remove the label and merge and cherrypick this PR?

**Comment by rh-tokeefe** [comment]
@bscott-rh sorry about that. I forgot to remove the label. Please merge and CP when you have a moment. Thanks!

**Pull request merged**